services/QuillLMS/app/models/admin_info.rb

```diff
@@ -45,9 +45,9 @@ class AdminInfo < ApplicationRecord
     BILLING_CONTACT = "Billing contact"
   ]
 
-  SUB_ROLES = SELECTABLE_SUB_ROLES.concat([
+  SUB_ROLES = SELECTABLE_SUB_ROLES + [
     TEACHER_ADMIN = 'Teacher admin'
-  ])
+  ]
```

Comment:
just curious about what the difference is between using `.concat([value])` and `+ [value]` ?

Reply:
Great question! `.concat` was modifying the original array, so both of them were returning the same list.